pkg/dryrun/policy.open-cluster-management.io_configurationpolicies.yaml

```diff
@@ -213,6 +213,54 @@ spec:
                         compared with objects on the cluster.
                       type: object
                       x-kubernetes-preserve-unknown-fields: true
+                    objectSelector:
+                      description: |-
+                        ObjectSelector defines the label selector for objects defined in the `objectDefinition`. If
+                        there is an object name defined in the `objectDefinition`, the `objectSelector` is ignored.
```

Comment:
Should we add this as a  test?

Reply:
"there is an object name defined in the `objectDefinition`, the `objectSelector` is ignored."

Reply:
Good call! I've added a test (and a code fix to address its failure! :sweat_smile:)